dissect/target/containers/fortifw.py

```diff
@@ -0,0 +1,206 @@
+import gzip
+import io
+import logging
+import os
+import zlib
+from itertools import cycle, islice
+from pathlib import Path
+from typing import BinaryIO, Optional, Sequence, Union
+
+from dissect.util.stream import RangeStream, RelativeStream
+
+from dissect.target.container import Container
+from dissect.target.tools.utils import catch_sigpipe
+
+logger = logging.getLogger(__name__)
+
+
+def find_xor_key(fobj: io.BytesIO) -> bytes:
```

Comment:
```suggestion
def find_xor_key(fh: io.BytesIO) -> bytes:
```
`fh` is a bit more consistent with other code.